content/en/hosting/kubernetes-vs-docker.md

```diff
@@ -2,48 +2,118 @@
+* A CHT API service.
+* HAProxy Healthcheck service.
+* HAProxy service.
+* Upgrade Service service.
+* A CHT Sentinel service.
+* An nginx service to act as a reverse proxy and terminate TLS connections
+
+
+## Kubernetes (k8s)
+
+{{< callout type="warning" >}}
+Medic recommends Kubernetes for bare-metal servers with a low core count or for multi-tenant deployments.
+{{< /callout >}}
+
+Running CHT on k8s requires additional knowledge and can be more complex to deploy. Docker is the preferred solution for most production deployments.
+
+k8s offers:
+
+* Resilient network across either physical or VM nodes in the cluster. This allows strong distribution of the heavy CPU and RAM loads that CouchDB can incur under heavy use.
 * Orchestration of multiple deployments ensuring to allow easy hosting of multi-tenants. For example you may opt to have a user acceptance testing (UAT), staging and production instances all in one cluster.
 * Service discovery which  enables to route public requests to deployments within the cluster
 * Integration with hypervisors such as VMWare and ESX. This is possible due to CRD support in Kubernetes that allows 3rd parties to create integrations
 * Helm support and integration that makes it easy to easily deploy applications on to the cluster
 * Highly efficient snapshot backups when used with a storage area network [SAN](https://en.wikipedia.org/wiki/Storage_area_network) or Amazon's [Elastic Block Storage](https://aws.amazon.com/ebs/) (EBS).
 
-The main components of a Kubernetes deployment include:
+The main components of a Kubernetes CHT deployment include one less service than a Compose based deployment as ingress via `nginx` isn't needed as it's handled externally:
 
 * A pod for each CouchDB instance.
 * A CHT API pod.
 * A CHT HAProxy Healthcheck pod.
 * A CHT HAProxy pod.
 * Upgrade Service pod.
 * CHT-Sentinel pod.
-## Docker Compose
 
-The Docker Compose based CHT Core deployment  was Medic's first attempt to make CHT 4.x cloud native.  The Compose files work quite well for application developer setups on laptops and the like (check out the [Docker Helper]({{< relref "hosting/4.x/app-developer#cht-docker-helper-for-4x" >}})!). Additionally, we have existing published guides on how to deploy single and multi-node Compose based solutions.  For small, low use instances, likely the single node Compose deployments will be fine.  We do not recommend setting up a multi-node deployment on Compose with an overlay network. Use Kubernetes instead for a more stable and [horizontally scalable]({{< relref "hosting/vertical-vs-horizontal" >}}) solution.
+## Example deployments
 
-Like Kubernetes above, Docker Compose deploys the same services but adds an additional 7th to allow for ingress/egress via a reverse proxy:
+Below are examples from 2 real world Ministries of Health (MoH) which have been anonymized and Medic hosted instances.  Users are total provisioned users, fewer users may be active in a given month.
```

Comment:
```suggestion
Below are examples from real world large-scale CHT deployments which have been anonymised.  Users are total provisioned users, fewer users may be active in a given month.
```

Reply:
I don't see any benefit in mentioning whether Medic is hosted or not. The important information here is the scale and technology used.

Reply:
Can we use average monthly active users instead of provisioned users?

Reply:
oop - missed Diana's request for average monthly active users - unresolving!

Reply:
moved all instances to 30 day average active user  in 986f4df8